k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```diff
@@ -92,6 +92,12 @@ spec:
                     name: CDM
                     recurse: true
                     title: CDM Builds Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api).+\/develop
```

Comment:
Well, this regex should rather pick the PR builds with specific pr numbers.
Eg. HMCTS_CDM/job/ccd-data-store-api/view/change-requests/job/PR-1260/